### ISSUE TITLE
Accessibility: Stepper

### ISSUE THREAD

**Issue by gfellerph** [swisspost/common-web-frontend#108]
![image](https://user-images.githubusercontent.com/1659006/170207872-a1dbf510-f406-4620-918e-4aed0a9bea73.png)

The stepper is missing semantics for sensible screen reader support. The progress is not recognised correctly and not read, step numbers are not read and completed steps (checkmark) are not recognized as well.

- [ ] Progress is not recognized by screen readers
- [ ] Step numbers are not read
- [ ] Completed steps are not recognized
- [ ] Total number of steps is not read

Inspiration: https://component.gallery/components/progress-indicator/?features=Accessibility
